Homework - 02 Basic Python.ipynb

```diff
@@ -11,11 +11,40 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 7,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Please enter the name of item you would like to buyBanana\n",
+      "Type 'done' to end shopping else type 'continue'continue\n",
+      "Please enter the name of item you would like to buySprite\n",
+      "Type 'done' to end shopping else type 'continue'continue\n",
+      "Please enter the name of item you would like to buyCoffee\n",
+      "Type 'done' to end shopping else type 'continue'done\n",
+      "['Banana', 2, 'Sprite', 2, 'Coffee', 5] Total price:  9\n"
+     ]
+    }
+   ],
    "source": [
-    "# your code"
+    "# your code\n",
+    "dict1={\"Coffee\":5,\"Sushi\":7.5,\"Sprite\":2,\"Banana\":2}\n",
+    "basket=[]\n",
+    "aa = 0\n",
+    "x = \"continue\"\n",
+    "while str(x) == 'continue':\n",
```

Comment:
`x` is already a string, no need to cast it (you don't need to use the `str` function).